integration/containerd/cri/integration-tests.sh

```diff
@@ -182,6 +182,11 @@ main() {
 
 	for t in "${passing_test[@]}"
 	do
+		# In case there is some lagacy containerd process left, 
+		# try to kill them.
+		if [ -f /run/containerd/containerd.sock ]; then
```

Comment:
It's good to check if there are things around we don't expect :-), but, are they a sign that something has gone wrong with an earlier test? If so, I'd rather we caught that and error'd out (like we do for stuck processes over in https://github.com/kata-containers/tests/blob/master/lib/common.bash#L80 for instance).
Would this be better added to `check_processes` so we can capture a broken test as early as possible (and not have to try and debug it after all other tests have run etc.)?